pink/tasks/joint_coupling_task.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright 2022 Stéphane Caron
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Joint coupling task implementation."""
+
+from typing import Sequence
+
+import numpy as np
+
+from ..configuration import Configuration
+from ..utils import get_joint_idx
+from .linear_holonomic_task import LinearHolonomicTask
+
+
+class JointCouplingTask(LinearHolonomicTask):
+    r"""Child class for linear holonomic tasks.
+
+    Note:
+        This task only considers a 1-Dimensional task.
```

Comment:
In general we could have a single task for multiple couplings, but it will be equivalent to multiple 1D tasks to going for this is fine by me too :ok_hand: